app/components/BadgesModal.tsx

```diff
@@ -184,11 +188,11 @@ export function BadgesModal({ msg = "View All Badges" }: { msg?: string }) {
 								<button
 									key={badge.filename}
 									onClick={() => setSelectedBadge(badge)}
-									className={`group flex flex-col items-center rounded-lg border bg-orange-500/50 p-2 text-white transition-all duration-200 ${
+									className={`group flex flex-col items-center rounded-lg border p-2 text-white transition-all duration-200 ${
 										selectedBadge?.filename === badge.filename
-											? "border-primary bg-orange-500/70"
-											: "border-primary/20 hover:border-primary/50 hover:bg-orange-500/60"
-									}`}
+											? "border-primary bg-primary-foreground/70"
+											: `border-primary/20 hover:border-primary/50 hover:${primaryForegroundSixtyOpacity}`
```

Comment:
Why are you mixing foreground color with background?

Reply:
In fact, here you should use a standard theme class of the button; `btn btn-primary` rather than adding custom classes. Similarly in other places.

Reply:
No, it's a badge card.